docs/beamrunner/beamrunner-3-sample.md

```diff
@@ -0,0 +1,187 @@
+
+By default, the graph shows the Beam transforms of the application and matches the pipeline structure that is described in the overview. The names that are used in the graph are the same as those used in the application unless there are transform names that are not unique.
+
+Although this simplified view matches Beam, it omits details about how  Streams is executing the application. To see those details, click  **Configure** (the wrench icon), select **Show raw graph**, and click **Apply**.
+
+Now the application is shown with the Streams operators instead of the Beam transforms. It is still similar in structure, and the names are similar to the Beam names but not always identical. In some cases, the Streams operator name contains extra information that comes from the names of the Beam Java™ SDK classes that implement the transforms. Also, Streams does not allow all the same characters in operator names that Beam allows in transform names, so characters might be removed or replaced with valid Streams characters.
+
+You can use the Beam [metrics API](https://beam.apache.org/documentation/sdks/javadoc/2.0.0/org/apache/beam/sdk/metrics/package-summary.html) to insert information to monitor your application. For more information, see [Monitoring IBM Streams Runner for Apache Beam](../beamrunner-4-monitor/).
+
+## Stopping the application
+
+After you explore the Streams console, stop the application by canceling the job. You can cancel the job in several ways, from almost any place that shows the job in the console, but the simplest way is by clicking **Cancel Jobs** in the navigation bar. Clicking **Cancel Jobs** opens a window that shows running jobs. Select the job or jobs that you want to cancel, in this case the job that is running the sample, and click **Cancel Jobs**.
+<img src="/streamsx.documentation/images/beamrunner/canceljobbutton.jpg" alt="Cancel Jobs in the navigation bar" width="100"/>
+
+<img src="/streamsx.documentation/images/beamrunner/canceljobwindow.jpg" alt="Cancel Jobs window" width="400" />
+
+**Important**: Interrupting or killing the `java` command in the terminal where you started the application does _not_ cause the job to be stopped, even though the job does stop reporting metrics.
+
+## The `TemperatureSample` application
+
+The `TemperatureSample` application takes temperature readings from multiple devices. The application splits the readings into "good" (valid) and "bad" (invalid) readings based on a specific threshold. It counts the bad readings and generates some basic statistics for the good readings, and finally logs the results. The full source is available in the `$STREAMS_RUNNER_HOME/samples/src/com/ibm/streams/beam/sample/temperature` directory, but excerpts are included here to describe the structure of the pipeline.
+
+The devices are artificial; they use the Beam `CountingInput` transforms named Counter\_*n* to drive another transform named Device\_*n* to generate random readings:
+
+```java

```

Comment:
It seems the java keyword here does not turn on the syntax highlighting in the doc. Any idea about what might be the reason?